### PR TITLE
Update faq slack info

### PR DESCRIPTION
**Pull request by crtrott**

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FAQ
 ###
 
-How do I join the Kokkos slack channel?
-  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We automatically whitelist emails from most organizations, but if your email address is not whitelisted, you can contact the Kokkos maintainers (their emails are in the LICENSE file).
+**How do I join the Kokkos slack channel?**
+  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack can invite more people. If no one you knows is in the Slack you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```
Comment (dalg24) [inline docs/source/faq.rst:5]:
```suggestion
  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack can invite more people. If no one you know is in the Slack you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```

**Comment by dalg24** [comment]
@crtrott should we just paste a permanent invite link they can just use?

**Review by nmm0 — APPROVED** [review]
Looks good after fixing the typo that damien pointed out

File: docs/source/faq.rst (hunk 1 above)
Comment (masterleinad) [reply docs/source/faq.rst:5]:
```suggestion
  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack workspace can invite more people. If no one you know is in the Slack workspace you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```

Comment (dalg24) [reply docs/source/faq.rst]: ignoring

Comment (masterleinad) [reply docs/source/faq.rst]: > [...] every member of the Slack can invite more people. 

Just doesn't sound correct.

File: docs/source/faq.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 FAQ
 ###
 
-How do I join the Kokkos slack channel?
-  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We automatically whitelist emails from most organizations, but if your email address is not whitelisted, you can contact the Kokkos maintainers (their emails are in the LICENSE file).
+**How do I join the Kokkos slack channel?**
+  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack can invite more people. If no one you know is in the Slack you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```
Comment (dalg24) [inline docs/source/faq.rst:5]:
```suggestion
  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Kokkos Slack workspace can invite more people. If no one you know is in the Slack workspace you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```

File: docs/source/faq.rst (hunk 1 above)
Comment (dalg24) [reply docs/source/faq.rst]: Updated

**Review by masterleinad — APPROVED** [review]

**Pull request merged**